scripts/vanilla.zs

```diff
@@ -166,13 +166,22 @@ recipes.addShaped("fire_resistance_long", <minecraft:potion>.withTag({Potion: "m
 recipes.addShaped("xp_bottle", <minecraft:experience_bottle>, [
     [<minecraft:potion>.withTag({Potion: "minecraft:water"}), <ore:dustSugar>, <ore:dustTinyGold>]
 ]);
+mixer.recipeBuilder()
+    .inputs(<minecraft:glass_bottle>)
+    .inputs(<ore:dustSugar>)
+    .inputs(<ore:dustTinyGold>)
+    .fluidInputs(<liquid:water> * 100)
+    .outputs(<minecraft:experience_bottle>)
+    .duration(30)
+    .EUt(14)
+    .buildAndRegister();
 
 // Nether Star
 chemical_reactor.recipeBuilder()
     .inputs(<ore:dustDiamond>)
     .inputs(<ore:dustIridium>)
     .fluidInputs([<liquid:rocket_fuel> * 1000])
-    .fluidInputs([<liquid:nether_air> * 8000])
+    .fluidInputs([<liquid:liquid_nether_air> * 8000])
```

Comment:
What was the reason for this change?

Reply:
just thought it fit better

Reply:
Liquid air here just makes this more annoying, and not any more difficult. Should really just use regular air.